apps/omg_watcher/lib/omg_watcher/supervisor.ex

```diff
@@ -32,8 +32,6 @@ defmodule OMG.Watcher.Supervisor do
   end
 
   def init(:ok) do
-    # prevent booting if contracts are not ready
-    :ok = RootChain.contract_ready()
```

Comment:
contracts cannot not be ready at this point, because we fetched contracts addresses from plasma 
 contracts in SetContracts module.